### PR TITLE
chore(ARCH-404): use stable storybook

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

we use rc of storybook

**What is the chosen solution to this problem?**

stable 6.4 has been released 6 days ago, let s use it.

**Please check if the PR fulfills these requirements**

- [x] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [x] Docs have been added / updated (for bug fixes / features)
- [x] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/storybook-cmf/package.json [hunk 1]
```diff
@@ -36,8 +36,8 @@
     "redux-saga": "^0.15.6"
   },
   "dependencies": {
-    "@storybook/addon-actions": "6.4.0-rc.3",
-    "@storybook/addons": "6.4.0-rc.3",
+    "@storybook/addon-actions": "^6.4.4",
+    "@storybook/addons": "^6.4.4",
```
Comment (gjou-tlnd) [inline packages/storybook-cmf/package.json:40]: doesn't make sens to have storybook deps in both deps & dev deps

Comment (jmfrancois) [reply packages/storybook-cmf/package.json]: the storybook plugin here depends on @storybook/addons but not addon-actions 
so I have removed it from here.
I dont remember why I have added @storybook/react as peer on this one but it is OK to have the same in peer and dev deps.
But none of them was in both deps and dev deps.

File: .changeset/beige-rats-hug.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@talend/react-components': minor
+---
+
+feat: make possible to project to upgrade virtualized list
```
Comment (frassinier) [inline .changeset/beige-rats-hug.md:5]: ?

**Review by frassinier — APPROVED** [review]

File: .changeset/beige-rats-hug.md (hunk 2 above)
Comment (jmfrancois) [reply .changeset/beige-rats-hug.md]: oups, this one was a missing file from previous PR 🤦 
Thanks to point it out.

**Review by gjou-tlnd — APPROVED** [review]

**Pull request merged**